modules/nw-configuring-router-compression.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+[id="nw-configuring-router-compression_{context}"]
+= Using router compression
+
+You configure the HAProxy Ingress Controller to specify router compression globally for specific MIME types. You can use the `mimeTypes` variable to define the formats of MIME types to which compression is applied. The types are: application, image, message, multipart, text, video, or a custom type prefaced by "X-". To see the full notation for MIME types and subtypes, see link:https://datatracker.ietf.org/doc/html/rfc1341#page-7[RFC1341].
+
+[NOTE]
+====
+Memory allocated for compression can affect the max connections. Additionally, compression of large buffers can cause latency, like heavy regex or long lists of regex.
+
+Not all MIME types benefit from compression, but HAProxy still uses resources to try to compress if instructed to.  Generally, text formats, such as html, css, and js, formats benefit from compression, but formats that are already compressed, such as image, audio, and video, benefit little in exchange for the time and resources spent on compression.
+====
+
+.Procedure
+
+. Configure the `httpCompression` field for the Ingress Controller.
+.. Use the following command to edit the `IngressController` resource:
++
+[source,terminal]
+----
+$ oc edit -n openshift-ingress-operator ingresscontrollers/default
+----
++
+.. Under `spec`, set the `httpCompression` policy field to `mimeTypes` and specify a list of MIME types that should have compression applied:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: default
+  namespace: openshift-ingress-operator
+spec:
+  httpCompression:
+    mimeTypes:
+    - "text/html"
+    - "text/css; charset=utf-8"
+    - "application/json"
+----
```

Comment:
Should you add the three dots `...` to indicate that there are other parameters in this file?

Reply:
See the 

In the [Code blocks, command syntax, and example output](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) section, a few lines down:
_For snippets or sections of a file, use an ellipsis (…​) to show that the file continues before and/or after the quoted block._

Reply:
Ok I think the ellipsis is good here, I added it.